src/main/kotlin/no/nav/familie/ef/mottak/mapper/ArkiverDokumentRequestMapper.kt

```diff
@@ -16,19 +19,30 @@ object ArkiverDokumentRequestMapper {
         val søknadsdokumentPdf =
                 Dokument(soknad.søknadPdf!!.bytes, FilType.PDFA, null, "hoveddokument", soknad.dokumenttype)
         val hoveddokumentvarianter = listOf(søknadsdokumentPdf, søknadsdokumentJson)
-        return ArkiverDokumentRequest(soknad.fnr, false, hoveddokumentvarianter, mapVedlegg(vedlegg))
+        return ArkiverDokumentRequest(soknad.fnr,
+                                      false,
+                                      hoveddokumentvarianter,
+                                      mapVedlegg(vedlegg, mapDokumenttype(soknad.dokumenttype)))
+    }
+
+    private fun mapDokumenttype(dokumenttype: String): String {
+        return when (dokumenttype) {
+            DOKUMENTTYPE_OVERGANGSSTØNAD -> DOKUMENTTYPE_OVERGANGSSTØNAD_VEDLEGG
+            DOKUMENTTYPE_BARNETILSYN -> DOKUMENTTYPE_BARNETILSYN_VEDLEGG
```

Comment:
Hva med arbeidssøkerskjema?

Reply:
... det skal ikke ha vedlegg - ergo skal vi ikke komme hit ... ? :)

Reply:
Takk, den har egentlige ikke noen vedlegg, men dette hadde feilet og i tillegg brukte jag ikke dokumenttypen for vedlegget til noe. Fikset og lagt til tester
https://github.com/navikt/familie-ef-mottak/pull/232/commits/ce6aceb1dc9ff8627db7da4ce4c1018082650ea7

Reply:
👍